testing/file_test/file_test_base.cpp

```diff
@@ -292,6 +296,7 @@ static auto MaybeApplyFileTestsFlag(llvm::StringRef factory_name) -> void {
   llvm::ListSeparator sep(":");
   for (const auto& file : absl::GetFlag(FLAGS_file_tests)) {
     filter << sep << factory_name << "." << file;
+    llvm::errs() << file << "\n";
   }
```

Comment:
Is this an intentional change or left-behind debug output? Given the function's comment I wouldn't expect it to write to stderr.